### DISCUSSION TITLE
Customizing color

### DISCUSSION THREAD

**Discussion by JanPalasek** [quarto-dev/quarto-cli#6428] [Q&A] [answered]
### Description

Hello.

Please how can I customize in the scss the color of clicking on a menu bar's item?

![Screenshot 2023-08-04 143119](https://github.com/quarto-dev/quarto-cli/assets/26582151/2f6736e0-b5c8-4f96-99f6-3bec9acf0052)

And how can I customize color of highlight in the searchbox? I suspect it is the same variable (that I didn't manage to find :-().

![Screenshot 2023-08-04 142926](https://github.com/quarto-dev/quarto-cli/assets/26582151/5f9601e6-fd94-4d9d-a7af-6c5ab44f5a1a)

The screenshots were taken from quarto.org website.

**Comment by mcanouil** [comment]
Use your browser developper mode ("inspect") to see what selector you need to modify, then modify it in you (S)CSS file.

**Comment by dragonstyle** [comment, marked as answer]
I believe that it is `$component-active-bg`... (defaults to `$primary$`).

**Reply by mcanouil** [reply]
The file with all the customisation from Quarto:

- https://github.com/quarto-dev/quarto-cli/blob/0995e8813d2774efbf92b3ba60c2bd2bf21188b0/src/resources/projects/website/search/quarto-search.scss

**Reply by JanPalasek** [reply]
Thanks. I modified primary in the end and it solved the issue.
